extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -118,27 +115,47 @@ function PanelMeasurementTableTracking({ servicesManager, extensionManager }) {
   };
 
   const createReport = async () => {
-    const measurements = MeasurementService.getMeasurements();
-    const trackedMeasurements = measurements.filter(
-      m =>
-        trackedStudy === m.referenceStudyUID &&
-        trackedSeries.includes(m.referenceSeriesUID)
-    );
+    const loadingDialogId = UIDialogService.create({
+      showOverlay: true,
+      isDraggable: false,
+      centralize: true,
+      // TODO: Create a loading indicator component + zeplin design?
+      content: () => <div className="text-primary-active">Loading...</div>
+    });
```

Comment:
This is interesting. I hadn't thought to do a whole screen `loading...`
Might be nice w/ a spinner, and setTimeout of `250ms` before clearing if the time has been less than `100ms` since this call (js Timer)

Reply:
The other option would be to replace the button's content, or text content near it, with a loading spinner.
A persistent notification until the event finishes would also be cool.
But I guess for all of these, we should have Dan weigh in.

Because we don't have a design here, can you create a JIRA follow-up ticket seeking clarification/guidance, explain the problem, your approach, and note a design/guidance would be appreciated. Assign Dan, move to current sprint.